openapi-generator/src/main/kotlin/org/jellyfin/openapi/builder/openapi/OpenApiApiServicesBuilder.kt

```diff
@@ -63,6 +64,39 @@ class OpenApiApiServicesBuilder(
 		else -> null
 	}
 
+	private fun buildRequestBody(requestBody: RequestBody, serviceName: String, operationName: String): TypeName? {
+		// Filter out duplicate JSON values
+		val requestBodyTypes = requestBody.content.keys.filterNot {
+			MimeType.IGNORED_JSON_TYPES.contains(it)
+		}
```

Comment:
Instead of filtering them out, they should maybe be mapped to the `application/json`, and stored in a set for deduplication. Imo that would be a bit cleaner.

Reply:
All the 3 json types are 100% the same content so we can just remove those other two

Reply:
If the main one is always passed, that's ok then.